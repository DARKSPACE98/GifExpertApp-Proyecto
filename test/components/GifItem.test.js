import { GifItem } from "../../src/components/GifItem";
import React, { Component } from 'react';
import { render } from "@testing-library/react";

describe('Pruebas en el GifItem', () => { 
    const title = 'Saitama';
    const url = 'https://one-punch.com/saitama.jpg';

    test('Titulo debe ser obligatorio', () => {
        
        const { container } = render(<GifItem title={title} />)
        expect(container).toMatchSnapshot();
     })

     test('Url debe ser obligatorio', () => {
        
        const { container } = render(<GifItem url={url} />)
        expect(container).toMatchSnapshot();
     })
     
 })