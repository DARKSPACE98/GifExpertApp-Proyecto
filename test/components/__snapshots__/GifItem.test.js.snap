// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el GifItem Titulo debe ser obligatorio 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="Saitama"
    />
    <p>
      Saitama
    </p>
  </div>
</div>
`;

exports[`Pruebas en el GifItem Url debe ser obligatorio 1`] = `
<div>
  <div
    class="card"
  >
    <img
      src="https://one-punch.com/saitama.jpg"
    />
    <p />
  </div>
</div>
`;
